docs/tutorials/index.rst

```diff
@@ -7,7 +7,7 @@ Browsing the tutorials
 The tutorials included with WaterTAP are available in the "tutorials" directory
 of the WaterTAP GitHub repository: `<https://github.com/watertap-org/watertap/tree/main/tutorials/>`_.
 
-To view a static version of the notebooks online, you can use the following link: `<https://nbviewer.org/github/watertap-org/watertap/blob/main/tutorials/00-index.ipynb>`_.
+To view a static version of the notebooks online, you can use the following link: `<https://nbviewer.org/github/watertap-org/watertap/blob/main/tutorials/00-index.ipynb>?flush_cache=true`_.
```

Comment:
```suggestion
To view a static version of the notebooks online, you can use the following link: `<https://nbviewer.org/github/watertap-org/watertap/blob/main/tutorials/00-index.ipynb?flush_cache=true>`_.
```

I'm not familiar with this option, but if `flush_cache=true` is supposed to be a URL parameter, then I believe it should be part of the URL in the ReST hyperlink.

Reply:
Thanks, @lbianchi-lbl . I realized this after the fact and committed your suggestion.